ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-2.4.yaml

```diff
@@ -114,6 +114,28 @@ tests:
           memory: 2Gi
       timeout: 20m0s
     workflow: servicemesh-istio-e2e-aws
+- as: update-charts-in-2-4
+  commands: |
+    # load test-infra automator.sh script
+    git clone https://github.com/maistra/test-infra.git
+    cd test-infra
+    export BUILD_WITH_CONTAINER="0"
+    export XDG_CACHE_HOME="/tmp/cache"
+    export GOCACHE="/tmp/cache"
+    export GOMODCACHE="/tmp/cache"
+    ./tools/automator.sh \
+      -o maistra \
+      -r istio-operator \
+      -b maistra-2.4 \
+      -f /creds-github/token \
+      '-c make clean update-charts-2.1 update-charts-2.2 update-charts-2.3' \
```

Comment:
Should you include 2.4 and remove 2.1?

Reply:
Do we have a 2.4 operator ready for update charts ? 
I can update that.